src/gtnh/models/available_assets.py

```diff
@@ -23,14 +23,16 @@ class AvailableAssets(GTNHBaseModel):
     def add_mod(self, mod: GTNHModInfo) -> None:
         log.info(f"Adding {mod.name}")
         bisect.insort_right(self.mods, mod, key=self._mod_sort_key)  # type: ignore
-        self._modmap.clear()
+        self.refresh_modmap()
 
     @staticmethod
     def _mod_sort_key(mod: GTNHModInfo) -> str:
         return mod.name.lower()
 
-    def clear_modmap(self) -> None:
-        self._modmap.clear()
+    def refresh_modmap(self) -> None:
+        # This is the correct way to reload a cached_property, but linter doesn't understand it whatsoever
+        # noinspection PyPropertyAccess
+        del self._modmap
```

Comment:
I think we need to guard this with a hasattr() check.  I was doing this but it would sometimes complain at runtime it couldn't find the attribute to delete